user/user_test.go

```diff
@@ -81,6 +81,7 @@ func makeRequest(env env, method string, url string, body string, authToken stri
 func makeMockEnv() env {
 	return env{
 		&mockDAO{userList: make([]dao.User, 0)},
```

Comment:
Didn’t we extract this `make` stuff away recently?

Reply:
We did... into this function ;)